<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="s1" duration-ms="7109" started-at="2022-05-02T18:40:45 CST" finished-at="2022-05-02T18:40:52 CST">
    <groups>
    </groups>
    <test name="t2" duration-ms="7109" started-at="2022-05-02T18:40:45 CST" finished-at="2022-05-02T18:40:52 CST">
      <class name="com.webtest.testCases.test1">
        <test-method status="PASS" signature="addListener(org.testng.ITestContext)[pri:0, instance:com.webtest.testCases.test1@5f375618]" name="addListener" is-config="true" duration-ms="12" started-at="2022-05-02T18:40:45 CST" description="æ·»å çå¬å¨" finished-at="2022-05-02T18:40:45 CST">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@5e81e5ac]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addListener -->
        <test-method status="PASS" signature="doBeforeClass()[pri:0, instance:com.webtest.testCases.test1@5f375618]" name="doBeforeClass" is-config="true" duration-ms="5806" started-at="2022-05-02T18:40:45 CST" finished-at="2022-05-02T18:40:51 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doBeforeClass -->
        <test-method status="FAIL" signature="test111()[pri:0, instance:com.webtest.testCases.test1@5f375618]" name="test111" duration-ms="45" started-at="2022-05-02T18:40:51 CST" finished-at="2022-05-02T18:40:51 CST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.webtest.testCases.test1.test111(test1.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.TestNG.privateMain(TestNG.java:1336)
at org.testng.TestNG.main(TestNG.java:1304)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test111 -->
        <test-method status="PASS" signature="doAfterMethod()[pri:0, instance:com.webtest.testCases.test1@5f375618]" name="doAfterMethod" is-config="true" duration-ms="1236" started-at="2022-05-02T18:40:51 CST" finished-at="2022-05-02T18:40:52 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doAfterMethod -->
      </class> <!-- com.webtest.testCases.test1 -->
    </test> <!-- t2 -->
  </suite> <!-- s1 -->
</testng-results>
